src/devices/sound/cdda.cpp

```diff
@@ -114,7 +114,7 @@ void cdda_device::pause_audio(int pause)
 uint32_t cdda_device::get_audio_lba()
 {
 	m_stream->update();
-	return m_audio_lba;
+	return m_audio_lba - (m_audio_samples / (CD_MAX_SECTOR_DATA / 4));
```

Comment:
@987123879113 
```suggestion
	return m_audio_lba - (m_audio_samples / (CD_MAX_SECTOR_DATA / MAX_SECTORS));
```

Reply:
The `4` here comes from `m_audio_samples = (CD_MAX_SECTOR_DATA*sectors)/4;`, not `MAX_SECTORS`